libcaf_net/CMakeLists.txt

```
@@ -18,7 +18,8 @@ set(LIBCAF_NET_SRCS
   src/socket.cpp
   src/socket_manager.cpp
   src/stream_socket.cpp
-)
+  src/scribe.cpp
+        )
```

Where are the extra whitespaces coming from?

```suggestion
)
```